static/css/index.css

```
@@ -350,6 +350,107 @@ body.neko-game-active [id$="-lock-icon"] {
 }
 
+/* Goodbye 后的 return 入口 Phase 2 改为 CAT1 占位表现，但继续复用原有
+   DOM/ID/事件协议与拖拽逻辑。 */
+.neko-idle-return-button-container {
+    pointer-events: auto;
+    touch-action: none;
+}
+
+.neko-idle-return-btn {
+    --neko-idle-return-size: clamp(96px, 14vw, 122px);
+    width: var(--neko-idle-return-size);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)